engine/src/hopeit/dataobjects/jsonify.py

```diff
@@ -2,140 +2,16 @@
-        """
-        if isinstance(payload, _ATOMIC_TYPES):  # immutable supported types
-            if key is None:
-                return json.dumps(payload)
-            return json.dumps({key: payload})
-        if isinstance(payload, _LIST_TYPES):
-            return "[" + ', '.join(Json.to_json(item, key=None) for item in payload) + "]"
-        if isinstance(payload, _MAPPING_TYPES):
-            return "{" + ', '.join(f'"{str(k)}": {Json.to_json(item, key=None)}' for k, item in payload.items()) + "}"
-        assert getattr(payload, 'to_json'), \
-            f"{type(payload)} should be annotated with @dataobject"
-        try:
-            return payload.to_json(validate=payload.__data_object__['validate'])  # type: ignore
-        except (ValidationError, AttributeError) as e:
-            raise ValueError(f"Cannot convert to JSON: type={type(payload)} validation_error={str(e)}") from e
-
-    @staticmethod
-    def to_obj(payload: EventPayloadType, key: Optional[str] = 'value') -> Union[dict, list]:
-        """
-        Converts event payload to dictionary or list
-
-        :param payload: EventPayload, instance of supported object type
-        :param key: key name used in generated json when serializing atomic values
-        :return: dict or list containing mapped representation of data. In case of simple datatypes,
-            a key:value form will be generated using key parameter. All objects mappable to dict will
-            be converted. Flat collections will be converted to list.
-        """
-        if isinstance(payload, _ATOMIC_TYPES):  # immutable supported types
-            if key is None:
-                return payload  # type: ignore  # only for recursive use
-            return {key: payload}
-        if isinstance(payload, _UNORDERED_LIST_TYPES):
-            return [Json.to_obj(v, key=None) for v in sorted(payload)]
-        if isinstance(payload, _LIST_TYPES):
-            return [Json.to_obj(v, key=None) for v in payload]
-        if isinstance(payload, _MAPPING_TYPES):
-            return {k: Json.to_obj(v, key=None) for k, v in payload.items()}
-        assert getattr(payload, 'to_dict'), \
-            f"{type(payload)} should be annotated with @dataobject"
-        try:
-            return payload.to_dict(validate=payload.__data_object__['validate'])  # type: ignore
-        except (ValidationError, AttributeError) as e:
-            raise ValueError(f"Cannot convert to dict: type={type(payload)} validation_error={str(e)}") from e
-
-    @staticmethod
-    def parse_form_field(field_data: Union[str, dict], datatype: Type[EventPayloadType],
-                         key: str = 'value') -> EventPayloadType:
-        """Helper to parse dataobjects from form-fields where encoding type is not correctly set to json"""
-        if isinstance(field_data, str):
-            return Json.from_json(field_data, datatype, key)
-        return datatype.from_dict(field_data)  # type: ignore
+    warnings.warn(
```

Comment:
"Usage" (singular)
"will be removed in a future version"
replace , (comma)  with . (dot)  + space.